compiler/src/ammtojs.ts

```diff
@@ -65,6 +65,13 @@ const assignableToJsText = (assignable: LPNode, indent: string) => {
     outText += assignable.get('variable').t
   } else if (assignable.has('value')) {
     outText += assignable.get('value').t
+    try {
+      const t = assignable.get('value').t
+      if (!/"/.test(t) && t !== 'true' && t !== 'false' && !/\./.test(t)) {
+        parseInt(assignable.get('value').t)
+        outText += 'n'
+      }
+    } catch (e) { }
```

Comment:
Since the kinds of values that can be assigned in AMM are so small, this works, even if it looks crazy. But it only works because right now you can't define integer constants that aren't `int64`. If we solve that, this breaks.

This may also need to be refactored from producing things like `2n` to producing `BigInt("2")` if Safari *still* doesn't have bigint literal support.